test/unit_tests/test_translation.py

```diff
@@ -315,6 +390,273 @@ def test_translate_operation_with_unique_params(
         )
 
 
+OQC_PULSE_PROPERTIES = json.dumps(
+    {
+        "braketSchemaHeader": {
+            "name": "braket.device_schema.pulse.pulse_device_action_properties",
+            "version": "1",
+        },
+        "supportedQhpTemplateWaveforms": {},
+        "ports": {
+            "channel_15": {
```

Comment:
Any particular reason the channels are numbered 15 and 11 whereas drive frames are 0 and 1?

Reply:
Thanks! Wasnt sure if this was arbitrary or a conscious choice! Can you check what is the channel for q1_drive and put that just for consistency?